### PR TITLE
adding in training env

### PR DESCRIPTION
**Pull request by m-juckes-pins**
## Describe your changes

Adding in training env within infra directory

## Type of change 🧩

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update
- [ ] Other (please explain in the description section above)

## Checklist before requesting a review

- [ ] I have performed a self-review of my own code
- [ ] I have double checked this work does not include any hardcoded secrets or passwords
- [ ] I have made corresponding changes to the documentation
- [ ] I have provided details on how I have tested my code
- [ ] I have referenced the ticket number above
- [ ] I have provided a description of the ticket
- [ ] I have included unit tests to cover any testable code changes

### CODE REVIEW

File: infrastructure/environments/training.tfvars [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+# sql_config = {
+#   admin = {
+#     login_username = "pins-odt-sql-training-appeals-bo"
+#     object_id      = "0fff01c6-b8bc-4b32-8f2d-1a1d122b7189"
+#   }
+#   sku_name    = "S0"
+#   max_size_gb = 250
+#   retention = {
+#     audit_days             = 120
```
Comment (m-juckes-pins) [inline infrastructure/environments/training.tfvars:79]: Will these values need changing or stick with same as PROD

File: infrastructure/environments/training.tfvars [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+#   public_network_access_enabled = false
+# }
+
+vnet_config = {
+  address_space             = "10.15.32.0/22"
+  apps_subnet_address_space = "10.15.32.0/24"
+  main_subnet_address_space = "10.15.33.0/24"
+}
+
+# web_app_domain = "back-office-appeals.planninginspectorate.gov.uk"
```
Comment (m-juckes-pins) [inline infrastructure/environments/training.tfvars:95]: Not sure on the domain?

back-office-appeals-training.planninginspectorate.gov.uk" ?

File: infrastructure/local.tf [hunk 3]
```diff
@@ -52,4 +52,5 @@ locals {
     blob_endpont = replace(replace(azurerm_storage_account.documents.primary_blob_endpoint, "https://", ""), "/", "")
   }
 
+  environment = var.environment == "training" ? "train" : var.environment
```
Comment (m-juckes-pins) [inline infrastructure/local.tf:55]: If env training will shorten to just train so doc storage account is 24 chars

File: infrastructure/environments/training.tfvars [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+      cs_team      = "863ffff7-b3c2-4c83-b82b-c2443cac96ae"
+      inspector    = "e7a0bcbc-aa39-496f-8b73-aa91e7577285"
+      legal        = "b7b4aff7-df95-42cc-840e-b3a3a59f4d0d"
+      pads         = "d8504f10-b4b8-4959-9165-686bf9501168"
+      read_only    = "6d323e05-322e-44d3-9f26-0c427b047ccb"
+    }
+  }
+
+  integrations = {
+    horizon_api_url               = "http://10.224.161.68:8000"
```
Comment (m-juckes-pins) [inline infrastructure/environments/training.tfvars:21]: test env

File: infrastructure/environments/training.tfvars [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+    fo_service_manager   = "pins-ag-odt-appeals-fo-service-manager-training"
+    data_service_manager = "pins-ag-odt-data-service-manager-training"
+    iap                  = "pins-ag-odt-iap-training"
+    its                  = "pins-ag-odt-its-training"
+    info_sec             = "pins-ag-odt-info-sec-training"
+  }
+}
+
+common_infra_config = {
+  network_name = "pins-vnet-common-prod-ukw-001"
```
Comment (m-juckes-pins) [inline infrastructure/environments/training.tfvars:54]: change to test

File: infrastructure/environments/training.tfvars [hunk 6]
```diff
@@ -0,0 +1,95 @@
...
+  network_rg   = "pins-rg-common-prod-ukw-001"
+}
+
+documents_config = {
+  account_replication_type = "GZRS"
```
Comment (m-juckes-pins) [inline infrastructure/environments/training.tfvars:59]: LRS

File: infrastructure/local.tf [hunk 7]
```diff
@@ -4,8 +4,8 @@ locals {
   primary_location   = "uk-south"
   secondary_location = "uk-west"
 
-  resource_suffix           = "${local.service_name}-${var.environment}"
-  secondary_resource_suffix = "${local.service_name}-secondary-${var.environment}"
+  resource_suffix           = "${local.service_name}-${local.environment}"
```
Comment (m-juckes-pins) [inline infrastructure/local.tf:7]: had to change these two to local too for the key vault and storage account

**Review by beejjacobs — APPROVED** [review]
LGTM!

**Pull request merged**